.github/workflows/bump_project_version.yml

```diff
@@ -0,0 +1,42 @@
+#############################################################################
+# GitHub Action to Bump Project Version.
+#
+# Workflow starts every day at the end of the day.
+#
+#############################################################################
+name: "Bump CS Version in REPO"
+on:
+    schedule:
+        - cron: '0 0 * * *'
+    workflow_dispatch:
+permissions:
+    contents: write
+    pull-requests: write
+jobs:
+    bump:
+        name: Bump CS version in REPO
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout the latest code
+              uses: actions/checkout@v2
+              with:
+                  token: ${{ secrets.GITHUB_TOKEN }}
+            - name: Get Version
+              id: VERSION
```

Comment:
why you need this id ?

Reply:
This id is needed to set `VAR` so we can use it as latest version and pass it as argument